.github/workflows/ci.yml

```diff
@@ -11,8 +11,6 @@ jobs:
       fail-fast: true
       matrix:
         include:
-          - python-version: "3.6"
```

Comment:
just wondering, why not use something like:
```yaml
      matrix:
        python-version: ["3.7", "3.8", "3.9"]
        nautobot-version: ["{{cookiecutter.min_nautobot_version}}", "stable"]
```

Reply:
I'll get that added for our upcoming efforts.